### PR TITLE
fxa-deviceId in signed cert (alt)

### PR DESCRIPTION
**Pull request by dannycoates**
An alternate implementation of #1166 (see https://github.com/mozilla/fxa-auth-server/pull/1166#discussion_r51185023)

### CODE REVIEW

**Comment by rfk** [comment]
assigning to @dannycoates to triage the appropriate next action on this

File: lib/routes/sign.js [hunk 1]
```diff
@@ -96,15 +97,25 @@ module.exports = function (log, isA, error, signer, db, domain) {
             duration: duration,
             generation: sessionToken.verifierSetAt,
             lastAuthAt: sessionToken.lastAuthAt(),
-            verifiedEmail: sessionToken.email
+            verifiedEmail: sessionToken.email,
+            deviceId: sessionToken.deviceId ?
+              sessionToken.deviceId.toString('hex') : null
```
Comment (rfk) [inline lib/routes/sign.js]: That reminds me, I owe you a plan for ensuring most devices actually have a device id...

File: lib/routes/sign.js [hunk 2]
```diff
@@ -96,15 +97,25 @@ module.exports = function (log, isA, error, signer, db, domain) {
...
-        }, reply)
+        )
+        .then(
+          function(certResult) {
+            log.activityEvent(
+              'account.signed',
+              request,
+              {
+                uid: uid,
+                account_created_at: sessionToken.accountCreatedAt
```
Comment (rfk) [inline lib/routes/sign.js]: While we're here, could you please log the device-id as part of this event as well?

**Comment by rfk** [comment]
r+ from me, modulo any naming tweaks you might do over in https://github.com/mozilla/fxa-auth-db-mysql/issues/125

**Comment by rfk** [comment]
(and a squashed commit)

**Comment by dannycoates** [comment]
r? for merge

**Comment by rfk** [comment]
awesomesauce

**Pull request merged**